test/built-ins/ArrayIteratorPrototype/next/args-mapped-expansion-after-exhaustion.js

```diff
@@ -1,7 +1,8 @@
 // Copyright (C) 2014 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 9.4.4.7 S22
+es6id: 9.4.4.7
+esid: sec-%arrayiteratorprototype%.next S22
```

Comment:
`S22` referred to "Step 22" here: https://www.ecma-international.org/ecma-262/6.0/index.html#sec-createmappedargumentsobject 

There is no longer a "Step 22", so `S22` (or `S*`) should all be omitted from `esid`